templates/cf-apache.service.mustache

```diff
@@ -10,6 +10,8 @@ PartOf=cfengine3.service
 Type=forking
 ExecStart={{{vars.sys.workdir}}}/httpd/bin/apachectl start
 ExecStop={{{vars.sys.workdir}}}/httpd/bin/apachectl stop
+# ENT-8209
+ExecStop=sleep 2
```

Comment:
Nice

Reply:
@Lex-2008  Don't forget the related change in core (where the units are pulled in and used by buildscripts)